lms/static/scripts/frontend_apps/components/SubmitGradeForm.tsx

```diff
@@ -211,6 +175,7 @@ export default function SubmitGradeForm({
     if (!result.valid) {
       setValidationMessageMessage(result.error);
       setValidationError(true);
+      return false;
```

Comment:
See my comments about the meaning of the boolean result not being immediately obvious.